### PR TITLE
480x320 pixel color screen folder

### PR DESCRIPTION
**Pull request by rotorman**
- Copies `sdcard\c480x272` to `sdcard/c480x320` to support different screen size
- Add new background image (and image source file) for wizards to suit screen size
- Update `readme.md` to suit

### CODE REVIEW

**Comment by pfeerick** [comment]
Has the EL18 been officially announced by Flysky yet? As it's not appropriate to be mentioning it publicly yet if that's not the case.

File: README.md [hunk 1]
```diff
@@ -6,23 +6,27 @@ This repo contains the SD Card contents for all supported EdgeTX Radios.
...
     - TX16s / TX16s mkII
     - T16
     - T18
     - Horus x10s
     - Horus x12s
-    - Generallly, most color screen radios
-- **c320x480.zip** (320x480 portrait orientation color screen radios)
-    - Flysky Nirvana NV14
-- **bw128x64.zip** (128x64 black and white screen radios)
+    - Generally, Most color screen radios...
```
Comment (pfeerick) [inline README.md:15]: This line wasn't didn't get pulled across properly on your merge

Comment (rotorman) [reply README.md]: Thanks, fixed!

**Comment by rotorman** [comment]
> did you essentially just copy the c480x272 folder across for the first commit with no other changes.
Yep, just two backgrounds changed for now.

**Comment by pfeerick** [comment]
Ok, sounds good. We should probably be thinking about global color and B&W folders next though, instead of making it so you now need to update stuff (i.e. widgets) in three different places if they work for all color targets. Then the screen specific folders only have to track the screen specific variations of resources.

**Comment by freshollie** [comment]
@pfeerick @rotorman is there a reason the new entries in the manifest (`sdcard.json`) now have duplicate ID's? This is the first time a non unique identifier for a row has been added. `nv14-` and `pl18-` appear twice.

There needs to be some unique way I can identify a row. If this change is going to also be made to the `fw.json` in the EdgeTX main firmware, then we also need some unique way to identify a firmware there.

**Comment by pfeerick** [comment]
That's a bug... they shouldn't be there. One minute please...

**Comment by pfeerick** [comment]
Should be fixed now by #d99ccf4ad18f7af730678e14e74739f2725c2522 ... and you are right, it should be unique identifiers in each case, and will match EdgeTX main repo `fw.json`.

**Pull request merged**